tombstone_thresholds.hh

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2020 ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <cinttypes>
+#include <numeric>
+
+struct tombstone_thresholds {
+    uint32_t tombstone_warn_threshold = std::numeric_limits<uint32_t>::max();
+    uint32_t tombstone_fail_threshold = std::numeric_limits<uint32_t>::max();
```

Comment:
nitpick: the `tombstone_` prefix from these attributes can be removed without any loss of context, and it will make a couple of lines shorter

Reply:
Yes